ui/ui-components/utils/contextHelper.tsx

```diff
@@ -0,0 +1,25 @@
+export type ContextValue = [Context: React.Context<any>, value: any];
+
+export const renderContextProvider = <T extends object>(
+  Context: React.Context<T>,
+  value: T,
+  children: React.ReactNode
+) => <Context.Provider value={value}>{children}</Context.Provider>;
+
+export const renderNestedContextProviders = (
```

Comment:
Since these are components, they should be capitalized.
```suggestion
export const RenderNestedContextProviders = (
```

Reply:
It's purposely not a component so that it doesn't render an unnecessary <NestedContextProvider> in the React inspector.